README.md

```diff
@@ -1,2 +1,97 @@
 # react-ga-onetrust-consent
-React implementation of OneTrust consent banner integrated with Google Analytics signals.
+
+React implementation of OneTrust consent banner integrated with Google Analytics signals. Initially built to support analytics-enabled University of Michigan Teaching & Learning applications, this logic using Google Analytics tracking alongside OneTrust consent verification is written to be properly standardized; We will aim to iterate on this project such that it could be shared with other UofM web apps that utilize React and Google Analytics. 
```

Comment:
Rather than `UofM`, use `UMich` if there's any chance this module will be used by other organizations or universities.  Otherwise, `U-M` is commonly used in communications, although it is ambiguous, too.

```suggestion
React implementation of OneTrust consent banner integrated with Google Analytics signals. Initially built to support analytics-enabled University of Michigan Teaching & Learning applications, this logic using Google Analytics tracking alongside OneTrust consent verification is written to be properly standardized; We will aim to iterate on this project such that it could be shared with other UMich web apps that utilize React and Google Analytics. 
```

Reply:
Sure, makes sense to me.